### PR TITLE
Metrics API doc fixup

### PR DESCRIPTION
**Pull request by joel-hamill**
## Description

https://jira.mesosphere.com/browse/DOCS-1816

## Urgency
- [ ] Blocker 
- [x] High
- [ ] Medium

## Requirements
- Test all commands and procedures.
- Build content [locally](https://github.com/dcos/dcos-docs#test-local) and test for formatting/links.
- Add redirects to [dcos-website/redirect-files](https://github.com/dcos/dcos-website#managing-redirects).
- Change all affected versions (e.g. 1.7, 1.8, and 1.9).
- See the [contribution guidelines](https://github.com/dcos/dcos-docs#contributing).

### CODE REVIEW

File: 1.9/administration/monitoring/metrics/index.md [hunk 1]
```diff
@@ -4,24 +4,24 @@ feature_maturity: preview
...
 
-  * Host: metrics about the specific node which is part of the DC/OS cluster. 
-  * Container: metrics about cgroup allocations from tasks running in Mesos or Docker containerizers. 
-  * Application: metrics about a specific application running inside the DC/OS Universal [container runtime](/docs/1.9/usage/containerizers/).
+  * **Host:** metrics about the specific node which is part of the DC/OS cluster. 
+  * **Container:** metrics about cgroup allocations from tasks running in Mesos or Docker containerizers. 
+  * **Application:** metrics about a specific application running inside the DC/OS Universal [container runtime](/docs/1.9/usage/containerizers/).
 
-The metrics component provides an HTTP API which exposes these three areas. 
+The [metrics HTTP API](/docs/1.9/monitoring/metrics/metrics-api/) exposes these areas.
```
Comment (karlkfi) [inline 1.9/administration/monitoring/metrics/index.md]: > The [Metrics API](/docs/1.9/monitoring/metrics/metrics-api/) exposes these areas.

File: 1.9/administration/monitoring/metrics/index.md [hunk 2]
```diff
@@ -34,102 +34,4 @@ Per-container metrics tags enable you to arbitrarily group metrics, for example
...
-|-------------------|------------------------------|
-| net_rx_bytes    | Bytes received. |
-| net_rx_dropped    | Packets dropped on receive.  |
-| net_rx_errors    | Errors reported on receive. |
-| net_rx_packets    |  Packets received.  |
-| net_tx_bytes    |  Bytes sent. |
-| net_tx_dropped    | Packets dropped on send  |
-| net_tx_errors    | Errors reported on send. |
-| net_tx_packets    | Packets sent. |
+For more information on which metrics are collected, see the Metrics [reference](/docs/1.9/administration/monitoring/metrics/reference/).
```
Comment (karlkfi) [inline 1.9/administration/monitoring/metrics/index.md]: > For more information on which metrics are collected, see the [Metrics Reference](/docs/1.9/administration/monitoring/metrics/reference/).

File: 1.9/administration/monitoring/metrics/metrics-api.md [hunk 3]
```diff
@@ -4,29 +4,35 @@ feature_maturity: preview
 menu_order: 1
 ---
 
-The Metrics API exposes node, container, and application metrics.
+You can use the Metrics API to periodically poll for data about your cluster, hosts, containers, and applications. You can then pass this data to a third party service of your choice to achieve informative charts, dashboards, and alerts.
```
Comment (karlkfi) [inline 1.9/administration/monitoring/metrics/metrics-api.md]: poll or stream?

Comment (karlkfi) [inline 1.9/administration/monitoring/metrics/metrics-api.md]: nevermind. there's no streaming in this api

File: 1.9/administration/monitoring/metrics/reference.md [hunk 4]
```diff
@@ -0,0 +1,108 @@
+---
+post_title: Metrics Statistics Reference
```
Comment (karlkfi) [inline 1.9/administration/monitoring/metrics/reference.md]: is this really the right title? aren't they just metrics, not metrics statistics... metrics attributes, maybe?

Comment (joel-hamill) [inline 1.9/administration/monitoring/metrics/reference.md]: `Metrics Reference` is probably fine.

File: 1.9/administration/monitoring/metrics/metrics-api.md [hunk 5]
```diff
@@ -4,29 +4,35 @@ feature_maturity: preview
...
 
 The Metrics API is backed by the [DC/OS Metrics component](/docs/1.9/overview/architecture/components/#dcos-metrics), which runs on all nodes in the cluster.
 
-For more information about using the Metrics API, see [Metrics](/docs/1.9/administration/monitoring/metrics/).
 
 
 ## Routes
 
-Access to the Metrics API is proxied through the Admin Router on each node using the following route:
+Access to the Metrics API is proxied through the Admin Router on each node.
```
Comment (guenter) [inline 1.9/administration/monitoring/metrics/metrics-api.md]: Afaik admin router is not involved here. You connect to localhost:61001 instead.

File: 1.9/administration/monitoring/metrics/reference.md [hunk 6]
```diff
@@ -0,0 +1,107 @@
...
+# Container
+
+The following per-container resource utilization metrics are collected.
+
+### CPU usage info
+   <!-- https://github.com/apache/mesos/blob/1.0.1/include/mesos/v1/mesos.proto -->
+   
+| Metric            | Description                  |
+|-------------------|------------------------------|
+| cpus_limit    | The number of CPUs allocated. |
```
Comment (guenter) [inline 1.9/administration/monitoring/metrics/reference.md]: number of CPU shares allocated

File: 1.9/administration/monitoring/metrics/reference.md [hunk 7]
```diff
@@ -0,0 +1,107 @@
...
+| cpus_system_time_secs    | Total CPU time spent in kernal mode in seconds. |
```
Comment (guenter) [inline 1.9/administration/monitoring/metrics/reference.md]: kernel

**Review by guenter — CHANGES_REQUESTED** [review]

File: 1.9/administration/monitoring/metrics/metrics-api.md (hunk 5 above)
Comment (guenter) [inline 1.9/administration/monitoring/metrics/metrics-api.md]: I understand this now. Admin router is in fact the thing that's listening on localhost:61001. I didn't know this implementation detail and assumed that the routes are available on the public master IP, which is how I usually interact with AR. I suggest we don't mention the fact that it's admin router and just say connect to :61001 on the agent.

Comment (joel-hamill) [inline 1.9/administration/monitoring/metrics/metrics-api.md]: @guenter https://github.com/dcos/dcos-docs/pull/1017/commits/3b087908771b3ee51d650610f9e3d03cded29534

File: 1.9/administration/monitoring/metrics/index.md [hunk 8]
```diff
@@ -4,24 +4,24 @@ feature_maturity: preview
...
 
-The Mesos Metrics Module is bundled with every agent in the cluster. This module enables applications to publish metrics from applications running on top of DC/OS to the collector by exposing a StatsD port and host environment variable inside every container. These metrics are appended with structured data such as agent-id, framework-id and task-id.
+The Mesos Metrics Module is bundled with every agent in the cluster. This module enables applications to publish metrics from applications running on top of DC/OS to the collector by exposing a StatsD port and host environment variable inside every container. These metrics are appended with structured data such as `agent-id`, `framework-id`, and `task-id`.
 
 <!-- insert graphic -->
 
-Per-container metrics tags enable you to arbitrarily group metrics, for example on a per-framework or per-system or agent basis. Here are the available tags:
+Per-container metrics tags enable you to arbitrarily group metrics, for example on a per-framework or per-system, or agent basis. Here are the available tags:
```
Comment (sascala) [inline 1.9/administration/monitoring/metrics/index.md]: suggest: "...for example, on a per-framework, per-system, or agent basis."

File: 1.9/administration/monitoring/metrics/index.md [hunk 9]
```diff
@@ -34,102 +34,4 @@ Per-container metrics tags enable you to arbitrarily group metrics, for example
 
 DC/OS applications will discover the endpoint via an environment variable (`STATSD_UDP_HOST` or `STATSD_UDP_PORT`). Applications leverage this StatsD interface to send custom profiling metrics to the system.
```
Comment (sascala) [inline 1.9/administration/monitoring/metrics/index.md]: s/leverage/use

File: 1.9/administration/monitoring/metrics/metrics-api.md [hunk 10]
```diff
@@ -4,29 +4,31 @@ feature_maturity: preview
...
+You can use the Metrics API to periodically poll for data about your cluster, hosts, containers, and applications. You can then pass this data to a third party service of your choice to achieve informative charts, dashboards, and alerts.
 
 The Metrics API is backed by the [DC/OS Metrics component](/docs/1.9/overview/architecture/components/#dcos-metrics), which runs on all nodes in the cluster.
 
-For more information about using the Metrics API, see [Metrics](/docs/1.9/administration/monitoring/metrics/).
-
-
 ## Routes
 
-Access to the Metrics API is proxied through the Admin Router on each node using the following route:
```
Comment (sascala) [inline 1.9/administration/monitoring/metrics/metrics-api.md]: "is the Metrics API resource path."

File: 1.9/administration/monitoring/metrics/metrics-api.md [hunk 11]
```diff
@@ -4,29 +4,31 @@ feature_maturity: preview
...
+For example, to view host-level metrics, run this command from your agent node:
 
-```
-/system/v1/agent/{agent_id}/metrics/v0/
+```bash
+curl -s http://localhost:61001/system/v1/metrics/v0/node | jq
 ```
 
-To determine the address of your cluster, see [Cluster Access](/docs/1.9/api/access/).
+From outside of the cluster, you can access by using this path, where `<resource-path>` is Metrics API resource path.
```
Comment (sascala) [inline 1.9/administration/monitoring/metrics/metrics-api.md]: "the Metrics API resource path."

**Review by sascala — COMMENTED** [review]
Some edits but otherwise LGTM

File: 1.9/administration/monitoring/metrics/metrics-api.md (hunk 11 above)
Comment (joel-hamill) [inline 1.9/administration/monitoring/metrics/metrics-api.md]: fixed

**Pull request merged**